### PR TITLE
Homework for Lesson 5

### PR DESCRIPTION
**Pull request by filkodaria**
Homework_5: task_1, task_2, task_3.

### CODE REVIEW

File: hw_5/hw_5.2.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+4. Написать скрипт, который преобразует любое предложение в camelCase. Первое слово должно начинаться с буквы в нижнем регистре, 
+  у остальных -  верхнем. Пример: I am super engineer => iAmSuperEngineer
+*/
+
+// subtask 1
+let initialBytes = 1456235400000;
+let divider = 1024;
+let result;
+
+if (initialBytes < 1024) {
```
Comment (anatoly-karpovich) [inline hw_5/hw_5.2.js]: Use while loop here)

File: hw_5/hw_5.2.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+
+// subtask 3
+let initialNumber = 100;
+let firstDivider = 3;
+let secondDivider = 5;
+
+for (let i = 1; i <= initialNumber; i++) {
+	if (i % firstDivider === 0 && i % secondDivider === 0) {
```
Comment (anatoly-karpovich) [inline hw_5/hw_5.2.js]:
```suggestion
	if (!(i % firstDivider) && !(i % secondDivider)) {
```

File: hw_5/hw_5.2.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+		console.log(`The number ${i} is divisible by ${firstDivider} and by ${secondDivider}.`)
+	} else if (i % firstDivider === 0) {
```
Comment (anatoly-karpovich) [inline hw_5/hw_5.2.js]: same usage as for 3 and 5

File: hw_5/hw_5.2.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+		console.log(`The number ${i} is divisible by ${firstDivider}.`)
+	} else if (i % secondDivider === 0) {
```
Comment (anatoly-karpovich) [inline hw_5/hw_5.2.js]: same usage as for 3 and 5

File: hw_5/hw_5.3.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+*/
+
+
+// subtask 1
+let initialWord = 'Alphabet';
+let vowels = 'aeiouy';
+let countVowels = 0;
+
+for (let i = 0; i < initialWord.length; i++) {
+	if (vowels.includes(initialWord[i].toLowerCase())) {
```
Comment (anatoly-karpovich) [inline hw_5/hw_5.3.js:24]: word can contain a "-" symbol, so use counter for consonants too

**Review by anatoly-karpovich — CHANGES_REQUESTED** [review]

File: hw_5/hw_5.1.js [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+// subtask 1
+let counter = 10;
+for (let i = counter; i >= 0; i--) {
+	console.log(i);
+}
+
+// subtask 2
+let rowAmount = 5;
+let pattern = ':)';
+let output = '';
```
Comment (VladimirLQA) [inline hw_5/hw_5.1.js]: Use `const` whenever possible

File: hw_5/hw_5.1.js [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+
+for (let i = 0; i < rowAmount; i++) {
```
Comment (VladimirLQA) [inline hw_5/hw_5.1.js:27]:
```suggestion
for (let i = 0; i < 5; i++) {
```
You can set condition without declaring variable / const

Comment (anatoly-karpovich) [reply hw_5/hw_5.1.js:27]: But its ok to have this-like depending on a variable)

Comment (VladimirLQA) [reply hw_5/hw_5.1.js:27]: It's okay, of course

Comment (filkodaria) [reply hw_5/hw_5.1.js:27]: @VladimirLQA yeah, I know it) 
but it seems to me to be easier managed if there is a variable outside the code block.
I would like to leave the such code view, if you are OK with that)

**Comment by filkodaria** [comment]
I've added changes to the files hw_5.1.js, hw_5.2.js, hw_5.3.js.
Please, check)

Still working on subtask 2 for Task 3 (can't implement negative offset).

File: hw_5/hw_5.2.js [hunk 8]
```diff
@@ -0,0 +1,130 @@
...
+	}
+	case 2: {
+		console.log(`The result is ${bytes.toFixed(1)} Mb.`);
+		break;
+	}
+	case 3: {
+		console.log(`The result is ${bytes.toFixed(1)} Gb.`);
+		break;
+	}
+	case 4: {
```
Comment (anatoly-karpovich) [inline hw_5/hw_5.2.js:60]: In case of greater initial number, counter value could be 5 or even more. You need to check if it is 4 or more, so gigabites

**Review by anatoly-karpovich — APPROVED** [review]
Good job! And please improve task 2-1 due to comments

**Comment by filkodaria** [comment]
I've added subtask 2 for task 3 (file hw_5.3.js). 
Please, add your advices for its implementation .

File: hw_5/hw_5.2.js (hunk 8 above)
Comment (filkodaria) [reply hw_5/hw_5.2.js:60]: changes done

**Review by VladimirLQA — APPROVED** [review]

**Pull request merged**